android_app/app/src/main/java/org/miniproject/safesweeper/MainActivity.java

```diff
@@ -58,22 +64,31 @@ protected void onCreate(Bundle savedInstanceState) {
         steeringToggle = (ToggleButton) findViewById(R.id.steeringToggle);
 
         new ConnectBT().execute();
+
         //toggle switch to go between manual steering and automatic steering, starts with manual
         steeringToggle.setOnClickListener(new View.OnClickListener() {
```

Comment:
Shouldn't this be in the ConnectBT onPostExecute? Since here we don't know if a connection has been established.

Reply:
Fixed, the .ino was not my intent to push (Have not touched it)! My condolences for this inconvenience.